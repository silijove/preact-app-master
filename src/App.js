import React from 'preact-compat';
import './App.css';

function App() {
    return (
        <div>
            <span>test</span>
        </div>
    );
}

export default App;
