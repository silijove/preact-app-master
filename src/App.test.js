import React, { render } from 'preact-compat';
import renderer from 'react-test-renderer';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  render(<App />, div);
});

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
