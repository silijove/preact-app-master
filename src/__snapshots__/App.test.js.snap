exports[`test renders correctly 1`] = `
<div
  __self={undefined}
  __source={
    Object {
      "fileName": "/home/rjs/workspaces/xebia/preact-app/src/App.js",
      "lineNumber": 7,
    }
  }
  className="App">
  <div
    __self={undefined}
    __source={
      Object {
        "fileName": "/home/rjs/workspaces/xebia/preact-app/src/App.js",
        "lineNumber": 8,
      }
    }
    className="App-header">
    <img
      __self={undefined}
      __source={
        Object {
          "fileName": "/home/rjs/workspaces/xebia/preact-app/src/App.js",
          "lineNumber": 9,
        }
      }
      alt="logo"
      className="App-logo"
      src="logo.svg" />
    <h2
      __self={undefined}
      __source={
        Object {
          "fileName": "/home/rjs/workspaces/xebia/preact-app/src/App.js",
          "lineNumber": 10,
        }
      }>
      Welcome to React
    </h2>
  </div>
  <p
    __self={undefined}
    __source={
      Object {
        "fileName": "/home/rjs/workspaces/xebia/preact-app/src/App.js",
        "lineNumber": 12,
      }
    }
    className="App-intro">
    To get started, edit 
    <code
      __self={undefined}
      __source={
        Object {
          "fileName": "/home/rjs/workspaces/xebia/preact-app/src/App.js",
          "lineNumber": 13,
        }
      }>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
